docs/BrowserObject.md

```diff
@@ -90,6 +90,18 @@ console.log(browser.config)
 console.log(browser.config.fakeUser) // outputs: "maxmustermann"
 ```
 
+### Versus Options
+
+Custom configurations should not be confused with [Options](Options.md), which are documented separately:
+
+```js
+browser.config.baseUrl; // Custom configuration
+browser.options.baseUrl; // WebdriverIO option
```

Comment:
This code example doesn't really make it clear for me, can you enhance it with some actual values and a snippet of a `wdio.conf.js`?

Reply:
Yep absolutely.

Reply:
I went ahead and added a few code snippets and clarified some of the language.